### PR TITLE
W-15874190-upgrade-rtfctl-appliance-pre-signed-url-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -178,6 +178,15 @@ rtfctl appliance upgrade --file runtimefabric-<VERSION>.tar.gz
 rtfctl appliance upgrade --url https://<host>/<path-to-installer-package>
 ----
 
+* Upgrades the cluster by using the installer presigned_url:
```
Comment (dmerlob) [inline modules/ROOT/pages/install-rtfctl.adoc:181]:
```suggestion
* Upgrades the cluster by using the installer `presigned_url`:
```

**Review by dmerlob — APPROVED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -178,6 +178,15 @@ rtfctl appliance upgrade --file runtimefabric-<VERSION>.tar.gz
...
+
+[NOTE]
+You must first obtain the installer pre-signed URL via the Runtime Manager platform API or using the following https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/runtime-fabric/minor/1.0/console/method/%232473/[Runtime Fabric API].
```
Comment (d4nielyan9) [inline modules/ROOT/pages/install-rtfctl.adoc:184]: Please change to https://anypoint.mulesoft.com/runtimefabric/api/download/installer-eol-extendedsupport 

Screenshot 2024-05-28 at 8 56 02 PM

Comment (d4nielyan9) [reply modules/ROOT/pages/install-rtfctl.adoc:184]: GET
```
curl --location 'https://anypoint.mulesoft.com/runtimefabric/api/download/installer-eol-extendedsupport' \
--header 'Authorization: Bearer <token>'
```

**Comment by kevintroller** [comment]
This will be created as Google doc internal file for certain customers.

**Pull request closed without merging**